learn/experimental/vector_search.mdx

```diff
@@ -5,14 +5,69 @@ description: Vector search is an experimental technology that uses Large Languag
 
-Are you using this feature? Meilisearch wants to hear from you! Tell us about your experience in the [GitHub discussion](https://github.com/meilisearch/product/discussions/621). All feedback is useful and helps make Meilisearch better and easier to use.
+Vector search is available to all users. [Meilisearch Cloud](https://www.meilisearch.com/cloud?utm_campaign=oss&utm_source=docs&utm_medium=vector-search) is the recommended way of using vector search.
 
-<Capsule intent="warning">
-This is an experimental feature. Experimental features are unstable: their API might significantly change and become incompatible between releases. Meilisearch does not recommend using experimental features in a production environment.
+## Using vector search
 
-Meilisearch makes experimental features available expecting they will become stable in a future release. However, it is not possible to guarantee when and if this will happen.
-</Capsule>
\ No newline at end of file
+### Activate vector search
+
+If using Meilisearch Cloud, navigate to your project overview and find "Experimental features", then check the "vector store" box.
+
+![A section of the project overview interface titled "Experimental features". There are two options: "Score details" and "Vector store". "Vector store" is turned on.](https://raw.githubusercontent.com/meilisearch/documentation/main/assets/images/vector-search/01-cloud-vector-store.png)
+
+Alternatively, use [the `/experimental` route](/reference/api/experimental_features) to activate vector search during runtime:
+
+```sh
+curl \
+  -X PATCH 'http://localhost:7700/experimental-features/' \
+  -H 'Content-Type: application/json'  \
+  --data-binary '{
+    "vectorSearch": true
+  }'
+```
+
+### Generate vector embeddings using a third-party tool
+
+To use vector search, you must generate embeddings with a third-party tool such as [OpenAI](https://openai.com/), [HuggingFace](https://huggingface.co/), [Cohere.AI](https://cohere.com/), and [Anthropic API](https://www.anthropic.com/).
+
+<Capsule intent="note" title="What is an embedding?">
+Vector search relies on embeddings. Vector embeddings are a way of representing data that identify similarities in meaning and context.
+</Capsule>
+
+### Add embeddings to Meilisearch
+
+Use [the `/documents` endpoint](/reference/api/documents) to upload vectorized documents. Store vector data in your documents' `_vector` field:
+
+```sh
+curl -X POST -H 'content-type: application/json' \
+'localhost:7700/indexes/products/documents' \
+--data-binary '[
+    { "id": 0, "_vectors": [0, 0.8, -0.2], "name": "frying pan" },
+    { "id": 1, "_vectors": [1, -0.2, 0], "text": "baking dish" },
+    { "id": 2, "_vectors": [[0.5, 3, 1], [-0.2, 4, 6]], "text": "spork" }
+]'
+```
+
+### Use `vector` to query Meilisearch
```

Comment:
@guimachiavelli I think we should also add in that section that it's possible to `filter` documents; someone asked us to add that feature while we already have it, it's just undocumented!

Reply:
Sure, no problem. How does that work exactly? Do we supply a vector to `filter`?

Reply:
No need for a vector to be passed to the filter. It's our standard filter expression. 

It's a way to refine / restrict the search results being given by the vector search.

Note: Most vector store competitors call document plain fields surrounding around the vectorized one `metadata`

Reply:
Ah, I see. I misunderstood what you said as "filtering a keyword-based search with a vector". Which might not make much sense, I guess.

So, to confirm, it would be something like this:

```sh
curl -X POST -H 'content-type: application/json' \
  'localhost:7700/indexes/products/search' \
  --data-binary '{ "vector": [0, 1, 2], "filter": "price < 10" }'
```